src/ripple/app/ledger/LedgerMaster.cpp

```diff
@@ -1117,8 +1125,16 @@ class LedgerMasterImp
                         InboundLedger::pointer acq =
                             getApp().getInboundLedgers ().findCreate (hash, seq, InboundLedger::fcGENERIC);
 
-                        if (!acq->isDone ())
+                        if (!acq)
                         {
+                            // On system shutdown, findCreate may return a nullptr
+                            WriteLog (lsTRACE, LedgerMaster)
+                                << "findCreate failed to return an inbound ledger";
+                            std::list<Ledger::pointer> emptyResult;
+                            return emptyResult;
```

Comment:
You can certainly replace these last two lines with:  `return std::list<Ledger::pointer>();` even in C++98.

But please try to see if in Windows uniform initialization works here:  `return {};`

There are a small number of cases in Windows where uniform initialization fails to compile - we're still sorting out what they are but I think this one's fine.